api/lanes.py

```diff
@@ -290,6 +289,7 @@ def lanes_for_large_collection(_db, languages):
              ),
             Lane(_db, full_name="Informational books", genres=None,
                  fiction=False, exclude_genres=[genres.Biography_Memoir],
+                 subgenre_behavior=Lane.IN_SAME_LANE,
```

Comment:
This was the cause of the errors that spurred #457. Because there were no genres, the `exclude_genres` set here was being ignored before (Biography/Memoir books can be see in the children's "Informational Books" lane at the moment). Now it's being properly excluded and the lane needs to be collapsed or else duplicate lane name errors ensue.